### PR TITLE
[Editor] provide reload button in the Siddhi editor console bar

### PR DESCRIPTION
**Pull request by SinthujanSintha**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/styles.css [hunk 1]
```diff
@@ -1799,4 +1799,21 @@ div.scrollbar > pre::-webkit-scrollbar-thumb {
...
+
+.isDisabled {
+    cursor: not-allowed;
+    opacity: 0.5;
+    color: #02aaad;
+}
+
+.isDisabled {
+    color: #02aaad;
+    display: inline-block;
```
Comment (suhothayan) [inline tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/styles.css]: Why we have 2 tags for isDisabled?

File: tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js [hunk 2]
```diff
@@ -217,21 +224,45 @@ define(['log', 'jquery', 'lodash', 'output_console_list', 'workspace', 'service_
...
-
             });
         return OutputConsoleList;
     });
+
+/**
+ *enable the reconnect button when  backend is unavailable.
+ * @param link
+ */
+let enableLink = function (link) {
```
Comment (suhothayan) [inline tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js]: dont use let

**Comment by niveathika** [comment]
@SinthujanSintha Any reason to have reload button is blue colour? Shall we have it similar to the clean button?

File: tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/styles.css (hunk 1 above)
Comment (SinthujanSintha) [reply tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/styles.css]: 13712b2ee95c1b9ae9345ac8be85cefbbe170fd9 this commit fix above requirement

File: tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js (hunk 2 above)
Comment (SinthujanSintha) [reply tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js]: 13712b2ee95c1b9ae9345ac8be85cefbbe170fd9 this commit fix above requirement

File: tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js [hunk 3]
```diff
@@ -217,21 +224,40 @@ define(['log', 'jquery', 'lodash', 'output_console_list', 'workspace', 'service_
...
+
+/**
+ *enable the reconnect button when  backend is unavailable.
+ * @param link
+ */
+var enableLink = function (link) {
+    link.removeAttribute('style');
+    link.classList.remove('console-sync-btn-disable');
+    link.href = link.getAttribute('data-href');
+    link.removeAttribute('aria-disabled');
```
Comment (niveathika) [inline tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js]: Move all style attributes to css class

**Comment by mohanvive** [comment]
@SinthujanSintha Can u respond to the comments

**Comment by niveathika** [comment]
@SinthujanSintha As discussed offline, since it's code maintainablity let's fix it later

**Pull request merged**